doc/model_persistence.rst

```diff
@@ -4,6 +4,47 @@
 Model persistence
 =================
 
+.. list-table:: Summary of model persistence methods
+   :widths: 25 50 50
+   :header-rows: 1
+
+   * - Persistence method
+     - Pros
+     - Risks / Cons
+   * - :ref:`ONNX <onnx_persistence>`
+     - * Serve models without a Python environment
+       * Serving and training environments independent of one another
+       * Most secure option
+     - * Not all scikit-learn models are supported
+       * Custom estimators are hard to add
+       * Original Python object is lost and cannot be reconstructed
+   * - :ref:`skops_persistence`
+     - * Pickle free and more secure
+       * Contents can be partly validated without loading
+     - * Not as fast as `pickle` based formats
+       * Supports less types than `pickle` based formats
+       * Requires the same environment as the training environment
+   * - :mod:`pickle`
+     - * Native to Python
+       * Can serialize most Python objects
+       * Efficient memory usage with `protocol=5`
+     - * Loading can execute arbitrary code
+       * No memory mapping
+       * Requires the same environment as the training environment
+   * - :mod:`joblib`
+     - * Efficient memory usage with memory mapping
+       * Easy shortcuts for compression and decompression
+     - * Pickle based format
+       * Loading can execute arbitrary code
+       * Requires the same environment as the training environment
+   * - `cloudpickle`_
+     - * Serialize custom Python code
+       * Comparable loading efficiency as :mod:`pickle`
```

Comment:
Cloudpickle uses the highest pickle protocol by default so it's efficient by default in that respect.

```suggestion
       * Comparable loading efficiency as :mod:`pickle` with `protocol=5`
```